docs/src/release-notes-js.md

```diff
@@ -6,6 +6,78 @@ toc_max_heading_level: 2
 
 import LiteYouTube from '@site/src/components/LiteYouTube';
 
+## Version 1.38
+
+### UI Mode Updates
+
+![Playwright UI Mode](https://github.com/microsoft/playwright/assets/746130/8ba27be0-58fd-4f62-8561-950480610369)
+
+1. Test timeline now supports range selection to zoom into specific test steps.
+1. Network panel revamp
+
+### New APIs
+
+- [`event: BrowserContext.webError`]
+- [`method: Locator.pressSequentially`]
+- The [`method: Reporter.onEnd`] now reports `startTime` and total run `duration`.
+
+### Deprecations
+
+* The following methods were deprecated: [`method: Page.type`], [`method: Frame.type`], [`method: Locator.type`], [`method: ElementHandle.type`]. Please use [`method: Locator.pressSequentially`] instead that provides clarity to test readers.
```

Comment:
We should point to `fill` instead of `pressSequentially`. Look at the deprecation message for `Locator.type` for inspiration.